LOAN/LoanTypes/MortgageLoans.rdf

```diff
@@ -152,57 +519,234 @@
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-typ-mtg;RemortgageOnDifferentTerms">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-mtg;Remortgage"/>
-		<rdfs:label xml:lang="en">Remortgage On Different Terms</rdfs:label>
+	<owl:Class rdf:about="&fibo-loan-typ-mtg;PropertyInspection">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-asmt;AssessmentActivity"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isProvidedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-pas-fpas;LegalAgent"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-mtg;isInspectionOf"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-plc-loc;RealEstate"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>property inspection</rdfs:label>
+		<skos:definition>assessment activity that involves analyzing one or more aspects of a real property for independent assessment of status or deficiency</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>This is independent and separate from conducting an appraisal. Examples are termite inspections, construction inspections, etc.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-typ-mtg;RemortgageWithEquityRelease">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-mtg;Remortgage"/>
-		<rdfs:label xml:lang="en">Remortgage With Equity Release</rdfs:label>
+	<owl:Class rdf:about="&fibo-loan-typ-mtg;PropertyUsage">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;Classifier"/>
+		<rdfs:label>property usage</rdfs:label>
+		<skos:definition>a category indicating the manner in which the borrower intends to utilize the property</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-typ-mtg;Renovation">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;MortgageLoanPurpose"/>
-		<rdfs:label xml:lang="en">Renovation</rdfs:label>
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;PropertyUsage-investment">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;PropertyUsage"/>
+		<rdfs:label>investment</rdfs:label>
+		<skos:definition>a home owned for the purpose of generating income</skos:definition>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;PropertyUsage-primaryResidence">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;PropertyUsage"/>
+		<rdfs:label>primary residence</rdfs:label>
+		<skos:definition>residence that the owner physically occupies and uses as his or her principal residence</skos:definition>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;PropertyUsage-secondHome">
+		<rdf:type rdf:resource="&owl;Thing"/>
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;PropertyUsage"/>
+		<rdfs:label>second home</rdfs:label>
+		<skos:definition>a property occupied by the owner for a portion of the year that is not the primary residence</skos:definition>
```

Comment:
does that include renting it out for remainder of the time e.g. AirBnB?

Reply:
No idea.